var/spack/repos/builtin/packages/libseccomp/package.py

```diff
@@ -18,15 +18,17 @@ class Libseccomp(AutotoolsPackage):
     version("2.5.3", sha256="59065c8733364725e9721ba48c3a99bbc52af921daf48df4b1e012fbc7b10a76")
     version("2.3.3", sha256="7fc28f4294cc72e61c529bedf97e705c3acf9c479a8f1a3028d4cd2ca9f3b155")
 
-    depends_on("c", type="build")  # generated
+    depends_on("c", type="build")
 
     variant("python", default=True, description="Build Python bindings")
 
     depends_on("gperf", type="build", when="@2.5:")
-    depends_on("py-cython", type="build", when="+python")
+
+    with when("+python"):
+        extends("python")
+        depends_on("py-cython", type="build")
+        # https://github.com/seccomp/libseccomp/commit/afbde6ddaec7c58c3b281d43b0b287269ffca9bd
+        depends_on("python@:3.11", type=("run", "link", "build"), when="@:2.5.4")
```

Comment:
```suggestion
        depends_on("python@:3.11", type=("run", "link", "build"), when="@:2.5.5")
```
2.5.5 still uses distutils, https://github.com/seccomp/libseccomp/blob/v2.5.5/src/python/setup.py#L26